docs/openstack/edpm_adoption.md

```diff
@@ -305,3 +294,179 @@ done
     ```
     oc wait --for condition=Ready osdpns/openstack --timeout=30m
     ```
+
+  ## Nova compute services fast-forward upgrade from Wallaby to Antelope
+
+> **NOTE**: Nova compute services upgrade cannot be performed in a lock-step with
+> Nova control plane services as those are managed independently by EDPM
+> ansible, and Kubernetes operators. The steps below implement that FFU
+> coordination instead, becaue there is yet a signalling interface between
+> different control loops that EDPM ansible and OCP operators utilize.
+
+* Define Nova compute services FFU config snippet for EDPM:
```

Comment:
bad news: that custom osdpservice  & osdpd does not trigger ansible jobs on my env.
good news: we ain't gonna need it after https://github.com/openstack-k8s-operators/nova-operator/pull/578